en/02_Developer_Guides/07_Debugging/01_Error_Handling.md

```diff
@@ -163,12 +163,19 @@ SilverStripe\Core\Injector\Injector:
   LogFileHandler:
     class: Monolog\Handler\StreamHandler
     constructor:
-      - "../silverstripe.log"
+      - "/var/www/silverstripe.log"
```

Comment:
`var/www/...` path depends on environment settings and it is relevant only if you use Linux system, but for Windows users (without Docker) it might be different. Could we mention about possible differences.